<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetCam1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1101</width>
      <height>891</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCam3" native="true">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>0</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCam2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>10</y>
      <width>371</width>
      <height>861</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_History">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>0</y>
      <width>421</width>
      <height>871</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>900</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetlLabel" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>900</y>
      <width>1281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
